source/common/stats/thread_local_store.cc

```diff
@@ -329,12 +328,62 @@ StatType& ThreadLocalStoreImpl::ScopeImpl::safeMakeStat(
   // If we have a TLS cache, insert the stat.
   if (tls_cache) {
     tls_cache->insert(std::make_pair((*central_ref)->statName(), *central_ref));
+    ENVOY_LOG_MISC(debug, "stat added to central cache map");
+    name.debugPrint();
+    ENVOY_LOG_MISC(debug, "central_cache_map = {}", static_cast<void*>(&central_cache_map));
+    ENVOY_LOG_MISC(debug, "this = {}", static_cast<const void*>(this));
+    ENVOY_LOG_MISC(debug, "parent_ = {}", static_cast<void*>(&parent_));
```

Comment:
I assume these will be removed before merging?

Reply:
Yup, this is still very much WIP :)